iphone/Classes/MediaModule.h

```diff
@@ -87,10 +87,9 @@
 @property (nonatomic, readonly) NSNumber *VIDEO_CONTROL_DEFAULT;
 @property (nonatomic, readonly) NSNumber *VIDEO_CONTROL_HIDDEN;
 
-@property (nonatomic, readonly) NSNumber *VIDEO_SCALING_NONE;
-@property (nonatomic, readonly) NSNumber *VIDEO_SCALING_ASPECT_FIT;
-@property (nonatomic, readonly) NSNumber *VIDEO_SCALING_ASPECT_FILL;
-@property (nonatomic, readonly) NSNumber *VIDEO_SCALING_MODE_FILL;
+@property (nonatomic, readonly) NSString *VIDEO_SCALING_MODE_RESIZE;
+@property (nonatomic, readonly) NSString *VIDEO_SCALING_MODE_RESIZE_ASPECT;
+@property (nonatomic, readonly) NSString *VIDEO_SCALING_MODE_RESIZE_ASPECT_FILL;
```

Comment:
Is `VIDEO_SCALING_MODE_RESIZE_ASPECT ` behaving differently than `VIDEO_SCALING_ASPECT_FIT` ? Same for `VIDEO_SCALING_MODE_RESIZE_ASPECT_FILL` and `VIDEO_SCALING_ASPECT_FILL `.